### PR TITLE
Ccp 1559 integrate bolt script

### PR DESCRIPTION
**Pull request by sznowicki**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

- [ ] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I updated the CHANGELOG.md

### CODE REVIEW

File: extension-config.json [hunk 1]
```diff
@@ -2,8 +2,73 @@
...
+      "id": "boltCartToken",
+      "path": "frontend/reducers/boltCartToken.js",
+      "type": "reducers"
+    }
   ],
   "configuration": {
+    "boldApiBaseUrl": {
+      "type": "admin",
+      "destination": "backend",
+      "default": "https://api-sandbox.bolt.com/",
```
Comment (Carsten89) [inline extension-config.json:26]: we should change this before doing the final version

File: extension-config.json [hunk 2]
```diff
@@ -2,8 +2,73 @@
...
+      "params": {
+        "required": true,
+        "type": "string",
+        "label": "Bolt Publishable Key"
+      }
+    },
+    "cdnUrl": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "connect-sandbox.bolt",
```
Comment (Carsten89) [inline extension-config.json:56]: we should change this before doing the final version

**Review by Carsten89 — APPROVED** [review]

File: extension-config.json (hunk 1 above)
Comment (sznowicki) [reply extension-config.json:26]: Added a ticket for that.

File: extension-config.json (hunk 2 above)
Comment (sznowicki) [reply extension-config.json:56]: Added a ticket for that.

**Pull request merged**